src/PrestaShopBundle/Bridge/Helper/Listing/HelperBridge/HelperListBridge.php

```diff
@@ -512,4 +503,32 @@ private function shouldLimitSqlResults($limit): bool
     {
         return $limit !== false;
     }
+
+    /**
+     * @param HelperListConfiguration $helperListConfiguration
+     * @param HelperList $helper
+     */
+    private function setHelperDisplay(
```

Comment:
This sounds more like `configureHelperList` from the look of it

Reply:
Yep it looks better indeed